lib/liquid/standardfilters.rb

```diff
@@ -234,6 +234,29 @@ def where(input, property, target_value = nil)
       end
     end
 
+    # Reject the elements of an array to those with a certain property value.
+    # By default the target is any falsy value.
+    def reject(input, properties, target_value = nil)
+      raise_property_error(properties) unless properties.is_a?(String)
+
+      properties = properties.to_s.split('.')
+      ary = InputIterator.new(input, context)
+
+      ary.reject do |item|
+        if item.is_a?(Hash)
+          value = item.dig(*properties)
+
+          if target_value.nil?
+            !value
+          else
+            value == target_value
+          end
+        else
+          true
+        end
+      end
+    end
```

Comment:
```suggestion
    def reject(input, properties, target_value = nil)
      raise_property_error(properties) unless properties.is_a?(String)

      properties = properties.to_s.split('.')
      array = InputIterator.new(input, context)
      check_array_inputs(array)
    end
    
    def check_array_inputs(array)
      array.reject do |item|
        if item.is_a?(Hash)
          value = item.dig(*properties)
          if target_value.nil?
            !value
          end
          return value == target_value
        else
          true
        end
      end
    end
```
```

Reply:
Just a few changes in the method, what do you think of this refactor? Good job

Reply:
Adding a new function here will add a new filter in Liquid. I don't think this change is necessary.